org-formation/650-identity-providers/_tasks.yaml

```diff
@@ -207,3 +207,38 @@ GithubOidcSageBionetworksScicompProvisioner:
   DefaultOrganizationBinding:
     Account: !Ref ScicompAccount
     Region: us-east-1
+
+GithubOidcSageBionetworksWebMonorepoInfra:
+  Type: update-stacks
+  DependsOn: GithubOidcSageBionetworks
+  Template: github-oidc-provider-access.njk
+  StackName: !Sub ${resourcePrefix}-${appName}-sage-bionetworks-web-monorepo-infra
+  Parameters:
+    GitHubOrg: "Sage-Bionetworks"
+    ProviderArn: !CopyValue [ !Sub '${resourcePrefix}-${appName}-ProviderArn' ]
+    PolicyDocument: |
+      {
+        "Version": "2012-10-17",
+        "Statement": [
+          {
+              "Sid": "ListObjectsInBucket",
+              "Effect": "Allow",
+              "Action": [ "s3:GetBucketLocation", "s3:ListBucket" ],
+              "Resource": [ "arn:aws:s3:::prod.accounts.sagebionetworks.org" ]
+          },
+          {
+              "Sid": "AllObjectActions",
+              "Effect": "Allow",
+              "Action": [ "s3:PutObject", "s3:GetObject" ],
+              "Resource": ["arn:aws:s3:::prod.accounts.sagebionetworks.org/*"]
+          }
```

Comment:
I think we will also need some sort Cloudfront role

Reply:
hmm, the policy you have now works for `s3 cp` command.   It looks like your script is doing a s3 sync so you might need some additional actions.  I'm thinking maybe you should add `s3:DeleteObject` and maybe `s3:*Multipart*` actions? 

It looks like your script is also doing a cloudfront create-invalidation so you'll want to add an allow for `cloudfront:CreateInvalidation` action.

Also your script is deploying to more than one bucket please add
`arn:aws:s3:::prod.accounts.sagebionetworks.org/*`

this might be it..
```
    PolicyDocument: |
      {
        "Version": "2012-10-17",
        "Statement": [
          {
              "Sid": "ListObjectsInBucket",
              "Effect": "Allow",
              "Action": [ "s3:GetBucketLocation", "s3:ListBucket", "s3:ListBucketMultipartUploads" ],
              "Resource": [ "arn:aws:s3:::prod.accounts.sagebionetworks.org" ]
          },
          {
              "Sid": "AllObjectActions",
              "Effect": "Allow",
              "Action": [ "s3:PutObject", "s3:GetObject", "s3:DeleteObject", "s3:*Multipart*" ],
              "Resource": ["arn:aws:s3:::prod.accounts.sagebionetworks.org/*", "arn:aws:s3:::staging.accounts.sagebionetworks.org/*" ]
          },
          {
              "Sid": "CloudfrontActions",
              "Effect": "Allow",
              "Action": [ "cloudfront:CreateInvalidation" ],
              "Resource": ["arn:aws:cloudfront::797640923903:distribution/E2656IE63W1MXI", "arn:aws:cloudfront::797640923903:distribution/E2656IE63W1MXI" ]
          }
        ]
      }
```

Reply:
Thanks @zaro0508 ! I have updated the PR to use this policy.